package-lock.json

```diff
@@ -79,9 +80,14 @@
         "typescript": "^5.0.4"
       }
     },
+    "apps/prompt-hack": {
+      "version": "0.0.1",
```

Comment:
Could you run `npm install` and commit the change if there is one. My environment is adding one small change to the lock file, just a `"name": "rebuff",` above this line.